cvat/apps/events/handlers.py

```diff
@@ -228,8 +252,16 @@ def _get_serializer(instance):
         serializer = CommentReadSerializer(instance=instance, context=context)
     if isinstance(instance, Label):
         serializer = LabelSerializer(instance=instance, context=context)
+    if isinstance(instance, Membership):
+        serializer = MembershipReadSerializer(instance=instance, context=context)
+    if isinstance(instance, Invitation):
+        serializer = InvitationReadSerializer(instance=instance, context=context)
+
+    return serializer
 
-    if serializer :
+def get_serializer(instance):
```

Comment:
It's pretty confusing to have both `get_serializer` and `_get_serializer` in the same module. Can't they have more distinctive names?